lighty-modules/lighty-bgp/src/test/java/io/lighty/modules/bgp/deployer/TestBase.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2021 PANTHEON.tech s.r.o. All Rights Reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at https://www.eclipse.org/legal/epl-v10.html
+ */
+
+package io.lighty.modules.bgp.deployer;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import io.lighty.core.controller.api.LightyController;
+import io.lighty.core.controller.api.LightyModule;
+import io.lighty.core.controller.api.LightyServices;
+import io.lighty.core.controller.impl.LightyControllerBuilder;
+import io.lighty.core.controller.impl.config.ConfigurationException;
+import io.lighty.core.controller.impl.config.ControllerConfiguration;
+import io.lighty.core.controller.impl.util.ControllerConfigUtils;
+import io.lighty.modules.bgp.config.BgpConfigUtils;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+abstract class TestBase {
```

Comment:
put all these `@BeforeAll` and `@AfterAll` methods to actual test class `ModuleInitTest`
extract these methods to "Base class" only when it is needed

Reply:
done

Reply:
done